stdlib/public/runtime/SwiftRT-WASM.cpp

```diff
@@ -0,0 +1,11 @@
+//===--- SwiftRT-WASM.cpp --------------------------------------------------===//
```

Comment:
I separated SwiftRT file to distinguish ELF and WASM.
No need to emit `__start` and `__stop` symbol because the runtime system which uses custom sections are disabled now and the latest clang raised runtime error to interpret `__asm__((__section__))`